examples/action_dann_lightn/model.py

```diff
@@ -5,12 +5,16 @@
 # Author: Haiping Lu & Xianyuan Liu
 # Initial Date: 7 December 2020
 
+import logging
 from copy import deepcopy
 
+import kale.pipeline.action_domain_adapter as action_domain_adapter
```

Comment:
Isn't `from kale.pipeline import action_domain_adapter` simpler?

Reply:
Yes, I will change all of them. This is the default style when `import` is added by PyCharm.